src/index.js

```diff
@@ -1 +1,37 @@
 'use strict';
+import express from 'express';
+import 'dotenv/config';
+import { authRouter } from './routes/auth.route.js';
+import cors from 'cors';
+import { userRouter } from './routes/user.route.js';
+import { errorMiddleware } from './middlewares/errorMiddleware.js';
+import cookieParser from 'cookie-parser';
+
+const PORT = process.env.PORT || 3000;
+
+const app = express(); // create web-server
+
+app.use(express.json());
+app.use(cookieParser());
+
+app.use(
+  cors({
+    // use cors to enter from different url
+    origin: process.env.CLIENT_HOST,
```

Comment:
Ensure that `process.env.CLIENT_HOST` is correctly set in your environment variables. If it's not set, CORS might not work as expected, potentially blocking requests from the client.